internal/rego/oci/oci.go

```diff
@@ -226,7 +226,7 @@ func ociImageManifest(bctx rego.BuiltinContext, a *ast.Term) (*ast.Term, error)
 		remote.WithAuthFromKeychain(authn.DefaultKeychain),
 	}
 
-	image, err := oci.NewClient(bctx.Context).Image(ref, opts...)
+	image, err := oci.NewClient(bctx.Context, opts...).Image(ref)
```

Comment:
Maybe just use the default opts here as well?

Reply:
Yeah